src/app.cpp

```diff
@@ -64,14 +65,15 @@ void transform(
 	tecmp_header header;
 	uint8_t* data;
 	int res = tecmp_next(packet_data, packet_header.captured_length, &iterator, &header, &data);
+	drop_replay_data = drop_replay_data && header.message_type == TECMP_TYPE_REPLAY_DATA;
```

Comment:
wrong logic here, create a new variable, otherwise not all replay will be dropped

Reply:
done